<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSearchClass</class>
 <widget class="QDialog" name="AdvancedSearchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>679</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>965</width>
    <height>654</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advanced Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>653</width>
       <height>175</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblByTitle">
        <property name="text">
         <string>By CaseNo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblByTitle_2">
        <property name="text">
         <string>By Title</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>By Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="txtEventNo"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtTitle"/>
      </item>
      <item row="1" column="2" colspan="4">
       <widget class="QLineEdit" name="txtDescription"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblBySpeaker">
        <property name="text">
         <string>By Speaker</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblLogNote">
        <property name="text">
         <string>Lognote</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="txtSpeaker"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cmbNote">
        <item>
         <property name="text">
          <string>Starts with</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ends With</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>contains</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2" colspan="4">
       <widget class="QLineEdit" name="txtLognote"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblLocation">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lblRoom">
        <property name="text">
         <string>Room</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="lblSearchEventType">
        <property name="text">
         <string>EventType</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="lblByDate">
        <property name="text">
         <string>By Date</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="txtLocation"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtRoom"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="txtEventType"/>
      </item>
      <item row="5" column="3">
       <widget class="QComboBox" name="cmbDate">
        <item>
         <property name="text">
          <string>After</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Before</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Between</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ignore</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QDateEdit" name="dateEditOne">
        <property name="displayFormat">
         <string>d MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QDateEdit" name="dateEditSecond">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>d MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="btnNewSearch">
        <property name="text">
         <string>New Search</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="4" colspan="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2" colspan="2">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="tblCases">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>280</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>290</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>CaseNo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date &amp; Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Speaker</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LogNotes</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" colspan="2">
       <widget class="QTableWidget" name="tblDetails">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>20</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Locations</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Room</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Event Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Start Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Case Status</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnCloseEvent">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Close Case</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnResumeEvent">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Resume Case</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="btnReopenEvent">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>ReOpen Case</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="btnViewNotes">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>ViewNotes</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="btnCancelResume">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>918</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancelResume</sender>
   <signal>clicked()</signal>
   <receiver>AdvancedSearchClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>932</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>965</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
